test/acceptance/tests/metrics/metrics_test.go

```diff
@@ -66,7 +66,7 @@ func TestComponentMetrics(t *testing.T) {
 	// This simulates queries that would be made by a prometheus server that runs externally to the consul
 	// components in the cluster.
 	logger.Log(t, "creating static-client")
-	k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.DebugDirectory, "../fixtures/cases/static-client-inject")
+	k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.DebugDirectory, "../fixtures/bases/static-client")
```

Comment:
We don't need to connect-inject static-client here because it's only used to connect to other pods. With tproxy, it doesn't work because all traffic goes through the proxy and we can't connect to those pods.